docs/dasharo-naming-convention.md

```diff
@@ -1,78 +1,78 @@
 # Dasharo Product Naming Convention
 
-Following documentation is results of [RFC](https://github.com/Dasharo/dasharo-issues/issues/762).
+Following documentation is v2 of naming convention following results of
+[RFC](https://github.com/Dasharo/dasharo-issues/issues/762).
 
 ## Synopsis
 
 ```plain
-Dasharo (firmware_framework[+payload]) [customer_type] Package for
-market_segment
+Dasharo (framework[+payload]) [edition] [release] Package for [platform]
```

Comment:
Right now we include `Dasharo (framework[+payload])` this part only in SMBIOS, I wonder if we should include more. Possibly related to: https://github.com/Dasharo/docs/pull/1154#issuecomment-3409588219 if we needed to tell which update given platform would need.